src/daemon/utils_cache.c

```diff
@@ -692,7 +693,8 @@ int uc_get_state(const data_set_t *ds, const value_list_t *vl) {
   return ret;
 } /* int uc_get_state */
 
-int uc_set_state(const data_set_t *ds, const value_list_t *vl, int state) {
+int uc_set_state(__attribute__((unused)) const data_set_t *ds,
```

Comment:
We could just remove the `ds` argument. There is only one call site.

Reply:
done